pkg/systemd/services/services.scss

```diff
@@ -95,3 +88,45 @@
 .pf-m-grid-md.pf-c-table [data-label] {
     display: revert;
 }
+
+// Magic pixie dust to make the list a bit faster and more furious
+.services-list {
+    // Prevent reflows; only applies to desktop (mobile changes display)
+    // table-layout: fixed;
+
+    tr {
+        // Skip a lot of computations
+        contain: content;
```

Comment:
Is this something that can be applied directly upstream to https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Table/table.scss ? 

I read little bit about these properties [here](https://css-tricks.com/almanac/properties/c/contain/) and they seem something generic enough to be applied upstream.

Reply:
I'm not sure, but it's possible... and it's a good idea; telling the browser to take some shortcuts when it can is always beneficial.

It can cause some rendering issues in some cases (depending on how others are using the lists), but at least _some_ setting of `contain` might be able to be a default and then could be set to something else for more or less optimization, depending on what's needed.

The primary win for this would be the tables, but any kinds of list or repeated and/or complex layout can see an improvement. (But it's usually negligible until there's a _lot_, so we probably shouldn't try to get this sprinkled everywhere.)